sklearn/preprocessing/data.py

```diff
@@ -2579,6 +2586,243 @@ def quantile_transform(X, axis=0, n_quantiles=1000,
                          " axis={}".format(axis))
 
 
+class PowerTransformer(BaseEstimator, TransformerMixin):
+    """Apply a power transform featurewise to make data more Gaussian-like.
+
+    Power transforms are a family of parametric, monotonic transformations
+    that are applied to make data more Gaussian-like. This is useful for
+    modeling issues related to heteroscedasticity (non-constant variance),
+    or other situations where normality is desired. Note that power
+    transforms do not result in standard normal distributions.
+
+    Currently, PowerTransformer supports the Box-Cox transform. Box-Cox
+    requires input data to be strictly positive. The optimal parameter
+    for minimizing skewness is estimated through maximum likelihood.
+
+    Read more in the :ref:`User Guide <preprocessing_transformer>`.
+
+    Parameters
+    ----------
+    method : str, (default='box-cox')
+        The power transform method. Currently, 'box-cox' (Box-Cox transform)
+        is the only option available.
+
+    copy : boolean, optional, default=True
+        Set to False to perform inplace computation during transformation.
+
+    Attributes
+    ----------
+    lambdas_ : array of float, shape (n_features,)
+        The parameters of the power transformation for the selected features.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.preprocessing import PowerTransformer
+    >>> pt = PowerTransformer()
+    >>> data = [[1, 2], [3, 2], [4, 5]]
+    >>> print(pt.fit(data))
+    PowerTransformer(copy=True, method='box-cox')
+    >>> print(pt.lambdas_)  # doctest: +ELLIPSIS
+    [ 1.051... -2.345...]
+    >>> print(pt.transform(data))  # doctest: +ELLIPSIS
+    [[ 0...      0.342...]
+     [ 2.068...  0.342...]
+     [ 3.135...  0.416...]]
+
+    See also
+    --------
+    power_transform : Equivalent function without the estimator API.
+
+    QuantileTransformer : Maps data to a standard normal distribution with
+        the parameter output_distribution='normal'.
```

Comment:
``output_distribution='normal'``

Reply:
Are you suggesting backticks? That's not obvious from the rendering ;)

Reply:
Ups ... thanks to point this out :)